e2e/e2e_test.sh

```diff
@@ -14,69 +15,44 @@ CI_MODE=${CI_MODE:-}
   # Backup current kubernetes context
 CURRENT_K8S_CTX=$(kubectl config view | grep "current" | cut -f 2 -d : | xargs)
 
-if GOOS="darwin"
-then
-  TIMEOUT_CMD=gtimeout
-else
-  TIMEOUT_CMD=timeout
-fi
+TIMEOUT_CMD=timeout
 
 
+# HELM BASED DEPLOYMENT
 
 # Prepare env
 kind create cluster --name "${KIND_CLUSTER_NAME}" --wait 5m
 kubectl config set-context kind-"${KIND_CLUSTER_NAME}"
 
-# Install prerequisites
-
-helm upgrade \
-  --install \
-  --repo https://charts.jetstack.io \
-  cert-manager cert-manager \
-  --namespace cert-manager \
-  --create-namespace \
-  --version v1.13.3 \
-  --set installCRDs=true \
-  --wait
-
-kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/download/v0.103.0/opentelemetry-operator.yaml --wait
-echo "Wait until otel operator pod is in ready state..."
-kubectl wait --namespace opentelemetry-operator-system --for=condition=available deployment/opentelemetry-operator-controller-manager --timeout=300s
+# Install telemetry-controller and opentelemetry-operator
+helm upgrade --install --wait --create-namespace --namespace telemetry-controller-system telemetry-controller oci://ghcr.io/kube-logging/helm-charts/telemetry-controller --version 0.0.10-dev.1
```

Comment:
I would recommend installing the chart from this repo instead as this doesn't test the current version

Reply:
This will be fixed before merging, since there is no proper semver tag of the chart, it wouldn't find the "latest" one.